### PR TITLE
[ASAbsoluteLayoutSpec] Change default value for ASAbsoluteLayoutSpecSizing

### PR DESCRIPTION
**Pull request by maicki**
This is under development and not for merge yet.

### CODE REVIEW

File: AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h [hunk 1]
```diff
@@ -12,10 +12,10 @@
 
 /** How much space the spec will take up. */
 typedef NS_ENUM(NSInteger, ASAbsoluteLayoutSpecSizing) {
-  /** The spec will take up the maximum size possible. */
-  ASAbsoluteLayoutSpecSizingDefault,
   /** Computes a size for the spec that is the union of all childrens' frames. */
   ASAbsoluteLayoutSpecSizingSizeToFit,
```
Comment (garrettmoon) [inline AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h]: Should we leave ASAbsoluteLayoutSpecSizingDefault but change it to mean ASAbsoluteLayoutSpecSizingSizeToFit?

File: AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
+  /** The spec will take up the maximum size possible. */
+  ASAbsoluteLayoutSpecSizingMaximum
```
Comment (appleguy) [inline AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h]: We could maybe call this SizeToParent, or SizeToFill?

SizeToChildren / SizeToParent, or SizeToFit / SizeToFill

**Comment by maicki** [comment]
For the transition phase I think we should close this PR and re-add static layout spec: #2426

**Comment by maicki** [comment]
Closing this pull request. If we have new insights for the default value it would be easier to open an ew PR.

**Pull request closed without merging**